chaos/scripts/check-case.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+completed=false
+for i in {1..20}; do
+    kubectl wait --for=condition=complete job/chaos-test-case --timeout=1m
+    if [ $? -eq 0 ]; then
+        completed=true
+        echo "chaos-test-case has completed"
+        break
+    else
+        echo "chaos-test-case has not completed" ${i}
+        kubectl get job chaos-test-case
+        if [ $? -ne 0 ]; then
+            echo "chaos-test-case job has been cleared"
```

Comment:
what's the cause of "cleared" 🤔

Reply:
If the test case job failed, K8s may clear the pod (and also the job) **sometimes** (I don't know how to disable this behavior yet, 😢 )